build.sh

```
@@ -1,4 +1,66 @@
+# Debug information
+echo "LD_LIBRARY_PATH: $LD_LIBRARY_PATH"
+ls -la $LIBSTDCXX_DIR/libstdc++.so* || true
+ldd $VIRTUAL_ENV/lib/python*/site-packages/pandas/_libs/*.so || true
+
+# Debug disk usage
+du -sh .
+top_usage=$(du -ah . | sort -rh | head -n 20)
+current_disk_usage=$(du -sm . | awk '{print $1}')
+echo -e "Current directory usage: ${current_dir_usage}M"
+echo -e "Top files/dirs usage: ${top_usage}\n"
+increment=$((current_disk_usage - initial_disk_usage))
```

Shellcheck (0.10.0)

[warning] initial_disk_usage is referenced but not assigned.

(SC2154)